test/rules/misused-new/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "misused-new": true
```

Comment:
let's call this `no-misused-new` to adhere better to existing naming conventions (we usually prefix "no-" to rules that lint _against_ the existence of some code)